.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: -- -A clippy::mutable_key_type
+          args: -- -A clippy::mutable_key_type -D warnings
```

Comment:
I understand your point but we want don't want to fail the build just because some rather opinionated lint by clippy is triggered, so warnings is sufficient IMHO.

//cc @maciejhirsz @dvdplm 

However, I think you can remove `-A clippy::mutable_key_type`

Reply:
I looked at fixing this clippy warning and was scratching my head tbh: it wasn't clear to me that the lint is valid (and I don't think it is in our case).

I'm fine keeping this as a warning in CI for the time being; github makes the clippy warnings fairly visible in the UI. And wee're still far away from a 1.0 release.

Reply:
At that point, there is not a single warning left. So it is now possible to be stricter, keep `-D warnings` and reject any new warnings. That is, with the option to allow them again later if some non sense lints shows up.

Do you still want to remove `-D warnings` nonetheless ?
Let me know and I will fix accordingly and remove `-A clippy::mutable_key_type` on the way.

Reply:
yeah, please remove `-D warnings` the warnings is still shown at every PR which is sufficient for now.

still great that you took care of this

Reply:
ok, will do 👍